db/019-change-alert-reason-values.sql

```diff
@@ -0,0 +1,44 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 19;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        -- Note: To view enum types and their values in `psql`, use the command `\dT+`
+        CREATE TYPE alert_type_enum AS ENUM ('BUTTONS_NOT_URGENT', 'BUTTONS_URGENT', 'SENSOR_NO_MOTION', 'SENSOR_DURATION', 'SENSOR_UNKNOWN');
```

Comment:
So I actually don't remember when we were coming up with the enum entries - do you remember why we opted for NO_MOTION over the extant STILLNESS?

Reply:
I think just because it's what was already being used in the Alert Lib's enum. I can change it though, it would make more sense based on the terminology that we're already using.

Reply:
Will be fixed in this new version of `brave-alert-lib`: https://github.com/bravetechnologycoop/brave-alert-lib/pull/24